ticket#10 contact us

daisyyedda opened the pull request:
finished designs

daisyyedda committed:
modified layout; added inclusivity statement

daisyyedda committed:
work in progress

daisyyedda committed:
finished contact us page

mahzabinrashid reviewed (changes requested):
Well done! This looks great. 

A few changes that need to be made:
1. The section isn't mobile-responsive. 
<img width="407" alt="Screenshot 2023-06-15 at 5 50 40 PM" src="https://github.com/itstechnova/technova-2023/assets/57790855/ee6ed880-b247-4575-83bd-cbb89167e811">

Could you make sure it looks good on mobile too?
2. Could you decrease some of the paddings when viewed on a laptop? (I'll tell you which ones in the meeting!)
3. Revert your changes added to `package.json`.
4. I feel like the email icon looks a little bigger than the rest of the icons. Could you please make it a little smaller?
5. There's a bug on the "code of conduct" button when you hover!
That's about it!

daisyyedda committed:
bugs fixed

daisyyedda commented:
a few things:
1. the page is now mobile responsive
2. changed some paddings to align with the ui
3. fixed the bug pertaining to "code of conduct" hyperlink
